.github/labeler.yml

```diff
@@ -4,9 +4,11 @@ component/zeebe:
       - '!.github/workflows/operate-*'
       - '!distro/**'
       - '!operate/**'
+      - '!operate.Dockerfile'
 component/operate:
   - changed-files:
     - all-globs-to-any-file:
       - '!.github/workflows/zeebe-*'
       - '!dist/**'
       - '!zeebe/**'
+      - '!Dockerfile'
```

Comment:
🤔 Actually there are more I think, might make sense to add these or we try the reverse.

- build-tools
- CHANGELOG.md
- clients
- CODE_OF_CONDUCT.md
- commitlint.config.js
- CONTRIBUTING.md
- dist
- Dockerfile
- docker-notice.txt
- license
- licenses
- maven-settings.xml
- monitor
- mvnw
- mvnw.cmd
- NOTICE.txt
- parent ?
- pom.xml
- README.md
- samples
- SECURITY.md
- spring-boot-starter-camunda-sdk
- target
- zeebe